lib/checkassert.cpp

```diff
@@ -37,8 +38,189 @@ static const struct CWE CWE398(398U);   // Indicator of Poor Code Quality
 // Register this check class (by creating a static instance of it)
 namespace {
     CheckAssert instance;
+
+    struct ArgumentCheck {
+        const std::vector<const Token *> *arguments;
+        const Scope *assertionScope;
+    };
+
+}
+
+
+/**
+ * \brief returns the lhs variable of the modification
+ * \param modifyOperator  - the modify operator
```

Comment:
I do not feel that this is a well written doxygen comment. 

The description for the parameter `modifyOperator` is:

    - the modify operator

I do not think it's proper to put that `-` at the start: